scripts/solve_network.py

```diff
@@ -517,17 +583,12 @@ def add_existing(n):
             add_existing(n)
 
         if (
-            snakemake.config["policy_config"]["monthly"]["reference_case"]
-            and eval(snakemake.wildcards["h2export"]) != 0
-            and snakemake.config["policy_config"]["policy"]
-            == "H2_export_monthly_constraint"
+            snakemake.config["policy_config"]["hydrogen"]["additionality"]
+            and snakemake.config["policy_config"]["hydrogen"]["temporal_matching"]
+            != "no_res_matching"

```

Comment:
Shouldn't also `is_reference` in config be false?

Reply:
That's right
Done